### DISCUSSION TITLE
Keyval resource - latest version of resource not found

### DISCUSSION THREAD

**Discussion by luis-n4489** [concourse/concourse#8685] [Help & Support]
Since I am doing a get on keyval resource (which still has not been updated with a version upon adding a new pipeline) it just stays stuck forever. I remember awhile back there was a way to force or curl to the resource via the api and set a dummy version / data? i dont remember if it was curl or the fly curl cmd. does anyone have the same issue or fix?

**Comment by taylorsilva** [comment]
`fly check-resource` is what you're looking for I believe. You can pass in a `--from key:val` flag.
